CHANGELOG.md

```diff
@@ -0,0 +1,30 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+**Note:** Version 0 of Semantic Versioning is handled differently from version 1 and above.
+The minor version will be incremented upon a breaking change and the patch version will be incremented for features.
+
+## [Unreleased]
+
+### Features : 
+- rpc: Creating a custom TPU Client instead of using solana TPU client more suitable for lite-rpc loads [PR](https://github.com/blockworks-foundation/lite-rpc/pull/105)
```

Comment:
```suggestion
- rpc: Creating a custom TPU Client instead of using solana TPU client more suitable for lite-rpc loads (https://github.com/blockworks-foundation/lite-rpc/pull/105)
```
Fine to just link to the PR, github renders the id